src/modals/createPageOrRecordModal.ts

```diff
@@ -42,6 +42,7 @@ export async function createPageOrRecordModal(
     modalInteraction: ModalInteractionStorage,
     tokenInfo: ITokenInfo,
     parent?: IDatabase,
+    page?:IPage,
```

Comment:
```suggestion
    page?: IPage,
```
Also this change. Else Everything LGTM!